vNext/shared/AppInsightsCommon/src/Util.ts

```diff
@@ -709,11 +709,11 @@ export class DateTimeUtils {
      * Gets duration between two timestamps
      */
     public static GetDuration = function (start: number, end: number): number {
-        var result = null;
-        if (start !== 0 && end !== 0 && !CoreUtils.isNullOrUndefined(start) && !CoreUtils.isNullOrUndefined(end)) {
-            result = end - start;
+        var duration = undefined;
+        if (!(isNaN(start) || isNaN(end))) {
+            duration = Math.max(end - start, 0);
```

Comment:
did you get a bug? I get it its safeguard code, but if end does turn out to be less than start, we should get that fbk about the capture, rather than reporting 0.